### PR TITLE
add extensions to composer platform

### PR DESCRIPTION
**Pull request by h-bragg**
Add the required extensions used (only for dev)

### CODE REVIEW

**Comment by biggianteye** [comment]
Tangent: Do you want to add `composer.lock`?

**Comment by biggianteye** [comment]
Some more tangents that are less immediate problems:
- The coverage tests don't pass because the echo server isn't running.
- `test-matrix` fails because of versioning (but is perhaps a moot point given travis runs multiple versions)
- Incorrect path: `-v ~/.composer:/root/composer`

File: composer.json [hunk 1]
```diff
@@ -10,6 +10,14 @@
     ],
     "homepage": "https://github.com/graze/dog-statsd",
     "license": "MIT",
+    "config": {
+        "platform": {
+            "ext-simplexml": "1.0",
```
Comment (biggianteye) [inline composer.json]: Minor observation: Although `simplexml` is listed as a required extension when this section does not exist, there are actually no complaints when this line is removed. I'm not sure why. If you see the same behaviour, consider simplifying by removing the line.

**Comment by h-bragg** [comment]
- added `echo` server to coverage methods.
- yeah, test-matrix will do that :(
- using official `composer` image, because we might as well...

File: composer.json (hunk 1 above)
Comment (h-bragg) [reply composer.json]: I guess xmlwriter implies simplexml?
who knows.

**Pull request merged**